packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -591,6 +591,11 @@
 
 @mixin bpk-table__cell {
   padding: $bpk-spacing-sm;
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    display: none;
```

Comment:
Why are we hiding the scrollbar?

Reply:
Good question - probably one for @matthewdavidson...

Reply:
It doesn't seem to actually have any effect...

Reply:
Did you try it in Safari?

Reply:
Ah - only tried it on mobile safari before, and it makes no difference there.
On desktop it does, but tbh I'm not sure it's the right thing to do. Maybe we should just leave it be? Scrollbars in MacOS are hardly invasive...

Reply:
Yeah I think leave it in - I'm concerned it'll harm a11y to remove them.

Reply:
Can we get design input into this please? We need to choose an option that is the best for all browsers / platforms. I.e. what does this look like on chrome for windows?

Reply:
AFAI can tell the `webkit-scrollbar` selector has no effect in edge.
Hiding the scroll bars in Safari and Chrome seems a little pointless as they are so unobtrusive anyway (ie they are invisible unless actively scrolling)

Reply:
> (ie they are invisible unless actively scrolling)

Have you checked this on windows?

Reply:
Yeah invisible scrollbars is a macOS / iOS thing afaik

Reply:
tbh - the more I think about this the more I think that the whole table should be scrolling, not the individual cell's content
Then we don't have to screw with scrollbars as the user / system setting will be approriate

![2019-03-15 09 43 11](https://user-images.githubusercontent.com/30267516/54422608-cad76380-4706-11e9-8e87-7eed13ebc591.gif)

Reply:
Perhaps we could use the mobile scroll container ^^

Reply:
Agreed (this was just a really quick gif I made in dev tools 😉)

Reply:
Dogfooding yo!
https://www.reddit.com/r/funny/comments/b0ksok/pure_betrayal/?utm_source=share&utm_medium=ios_app